actions/cli.yaml

```diff
@@ -0,0 +1,42 @@
+---
+name: "cli"
+runner_type: "python-script"
+description: "Run CLI commands on a device using NAPALM."
+enabled: true
+entry_point: "cli.py"
+parameters:
+    hostname:
+        type: "string"
+        description: "The hostname of the device to connect to. Driver must be specified if hostname is not in configuration. Hostname without FQDN can be given if device is in configuration."
+        required: true
+        position: 0
```

Comment:
Position argument is used only for scripts that that take positional arguments. In this case, cli.py seems like a python action which takes in named parameters.